src/main/java/at/hannibal2/skyhanni/features/garden/visitor/GardenVisitorTimer.kt

```diff
@@ -154,8 +155,8 @@ class GardenVisitorTimer {
             else -> "e"
         }
 
-        val extraSpeed = if (diff in 2.seconds..10.seconds) {
-            val duration = millis / 3
+        val extraSpeed = if (GardenAPI.isCurrentlyFarming()) {
+            val duration = (millis / 3) * (GardenCropSpeed.averageBlocksPerSecond / 20)
```

Comment:
averageBlocksPerSecond is only an average over a long time . we should rather use average over last 2 seconds or something

Reply:
Good point, I wasn't sure.